src/pages/TeamPage/TeamPage.tsx

```diff
@@ -201,6 +228,18 @@ export class TeamPage extends React.Component<ITeamPageProps, ITeamPageState> {
       <Section>
         <Heading><RedWord>Course</RedWord> Awareness</Heading>
       </Section>
+
+      <Section backgroundColor="white">
+        <FlexRow justifyContent="flex-end">
+          <FlexCell flex="0">
```

Comment:
`flex="0"` probably not necessary.
https://css-tricks.com/snippets/css/a-guide-to-flexbox/

It's treated as a ratio. For example:
```
<FlexRow>
  <FlexCell flex={1} />
  <FlexCell flex={2} />
</FlexRow>
```
You sum all `flex` values (3). The second `FlexCell` would be 2/1 wider.

If they were 2 and 3, then the second `FlexCell` would be 3/2 wider than the first